ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/nasm/tasks/main.yml

```diff
@@ -71,43 +71,51 @@
   tags: nasm
 
 - name: Download nasm sources
-  shell: cd /tmp && wget https://www.nasm.us/pub/nasm/releasebuilds/2.13.03/nasm-2.13.03.tar.gz
+  shell: cd /tmp && wget https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03.tar.gz
   when:
-    - (nasm_installed.rc is defined) and ((nasm_installed.rc != 0 ) or (nasm_installed.rc == 0 and nasm.stdout is version_compare('2.13.03', operator='ne')) )
+    - (nasm_installed.rc is defined) and ((nasm_installed.rc != 0 ) or (nasm_installed.rc == 0 and nasm.stdout is version_compare('2.16.03', operator='ne')) )
   tags: nasm
 
 - name: Extract nasm sources
   unarchive:
-    src: /tmp/nasm-2.13.03.tar.gz
+    src: /tmp/nasm-2.16.03.tar.gz
```

Comment:
Feels like the version could get extracted.

Reply:
Updated. Thanks!